### ISSUE TITLE
Add Project = shows up on the screen/view I'm on now...

### ISSUE THREAD

**Issue by jsonin** [goinvo/staffplan-next-app#112]
I'm in my staffplan.
I add a new project.
That new project row should show up in my staffplan.
Also, we need an empty project row on all individual staffplan screens.

Right now, to see the new project I just added, I have to go to the "Projects" screen. Hey, it saved!

**Comment by zhukovdigital** [comment]
@jsonin , So, if we create a new project through the new modal of creating project, we should immediatly assign this project to the user which is a viewer, correctly?

**Comment by jsonin** [comment]
Yes, I think that feels natural… assign to the person I’n currently looking at.We may find that to be weird in the future, but I think that works… for now.On Sep 6, 2024, at 2:18 PM, Oleksandr Zhukov ***@***.***> wrote:﻿
@jsonin , So, if we create a new project through the new modal of creating project, we should immediatly assign this project to the user which is a viewer, correctly?

**Comment by jsonin** [comment]
Here is an example of a painful screen:
<img width="1277" alt="image" src="https://github.com/user-attachments/assets/8ccc809e-17ee-4575-ac86-a1ebe868df55">
I go to a StaffPlan, Claire's in this case, and I can't do anything.
We need to have the blank project row here to start adding projects right on the main screen.
